bindings/python/google_benchmark/benchmark.cc

```diff
@@ -113,7 +113,14 @@ PYBIND11_MODULE(_benchmark, m) {
           "complexity",
           (Benchmark * (Benchmark::*)(benchmark::BigO)) & Benchmark::Complexity,
           py::return_value_policy::reference,
-          py::arg("complexity") = benchmark::oAuto);
+          py::arg("complexity") = benchmark::oAuto)
+      .def("setup",
+           (Benchmark * (Benchmark::*)(benchmark::State&)) & Benchmark::Setup,
```

Comment:
This still seems     ... wrong? 
@dominichamon  @LebedevRI

Reply:
P.S: actually i've figured it out - the explicit cast was wrong. (and it's not needed here anyway)